cime_config/buildnml

```diff
@@ -1470,6 +1479,8 @@ cat >>! $RUNDIR/ocn_in$inststr << EOF
   INID14C      = $INID14C
   WITH_DMSPH   = $WITH_DMSPH
   PI_PH_FILE   = $PI_PH_FILE
+  L_SED_POR    = $L_SED_POR
+  SEDPORFILE   = $SEDPORFILE
 /
 
```

Comment:
GitHub won't let me add a comment in an untouched part of the code, but further below at line 1840 there is a file `blom.input_data_list` created. If L_SED_POR is activated and SEDPORFILE specified, the filename should be added to this file (used by the CESM scripting system to check at submission whether all files are available)

Reply:
I see. Thanks for the explanation! I'll add that.